tests/nested/manual/split-refresh/task.yaml

```diff
@@ -0,0 +1,224 @@
+summary: Check app refreshes don't wait for a reboot in hybrid systems
+
+details: |
+  Check that (on hybrid systems) refreshes of both essential (e.g., kernel,
+  gadget) and non-essential snaps don't make apps and non-model bases wait for
+  a reboot like the essential snaps. Check that if an app depends on the model
+  base, it does wait for its refresh, and consequently, a reboot.
```

Comment:
Yes but not because of the base itself, because the kernel and gadget snap are updating and they depend on the base. So the base refreshing isn't causing the reboot, but it does need to be refreshed before the kernel/gadget because they use bits of it to run hooks AFAIK. I can refresh this summary to be "if an app depends on the same base as the updating kernel and gadget snaps" which maybe conveys better that the kernel/gadget are causing the reboot, not the base. wdyt?